ADR/0013-integration-service-api-contracts.md

```diff
@@ -93,11 +97,27 @@ Example for a run with an error:
     "result": "ERROR",
     "timestamp": "1649842004",
     "successes": 0,
+    "note": "Task fbc-validation failed: $(workspaces.source.path)/hacbs/inspect-image/image_inspect.json did not generate correctly. For details, check Tekton task result HACBS_TEST_OUTPUT in task inspect-image.",
     "failures": 0,
     "warnings": 0
 }
 ```
 
+
+#### Tekton Result Format for `CLAIR_SCAN_RESULT`
+The Test output of the Tekton result **CLAIR_SCAN_RESULT** will be a JSON object that includes the following information about the found vulnerabilities scanned by Clair. Refer to [Red Hat Vulnerability documentation](https://access.redhat.com/articles/red_hat_vulnerability_tutorial) for more context on the vulnerability severity ratings.
```

Comment:
I know that we decided to settle on `CLAIR_SCAN_RESULT` in the past, but I wonder if it would actually be more beneficial to generalize it further to something like `VULNERABILITY_SCAN_RESULT`. This type of result could be useful for other scanners to have their results displayed in the UI as well if the UI can generalize it. 

This would be beneficial for partner tasks as well.

Reply:
Thanks for your thought. Currently we have this additional result for Clair scan only and this PR to reflect the result change, so how about updating to a new name in the future when we have a requirement and chance to define more new result for partner tasks?

Reply:
Yeah, that makes sense. I had been thinking the same thing as well after I posted this comment.

Reply:
Thanks!